### ISSUE TITLE
Unable to set up VoxelStreamSqlite

### ISSUE THREAD

**Issue by LVLRGames** [Zylann/solar_system_demo#8]
when setting up sqlite as a stream for a VoxelLODTerrain, I am running into errors, and cant get any generation to happen either in the editor or runtime game. 

E 0:00:03:0967   zylann::voxel::VoxelStreamSQLite::load_voxel_blocks: Condition "con == nullptr" is true.
  <C++ Source>   modules\voxel\streams\sqlite\voxel_stream_sqlite.cpp:730 @ zylann::voxel::VoxelStreamSQLite::load_voxel_blocks()

E 0:00:03:0967   zylann::voxel::LoadBlockDataTask::run: Error loading voxel block
  <C++ Source>   modules\voxel\engine\load_block_data_task.cpp:64 @ zylann::voxel::LoadBlockDataTask::run()

E 0:00:03:0967   zylann::voxel::VoxelStreamSQLite::load_instance_blocks: Condition "con == nullptr" is true.
  <C++ Source>   modules\voxel\streams\sqlite\voxel_stream_sqlite.cpp:819 @ zylann::voxel::VoxelStreamSQLite::load_instance_blocks()
  
  I am using a voxel graph generator, which now works fine when there is no stream setup (thanks for the quick fix btw).  
  
![image](https://user-images.githubusercontent.com/87674745/223571384-f9f28563-454d-49f0-bc1b-17ac41e055d4.png)

I am attempting to save to the path saves/db/,  but I am not particularly sure of how to format the path or where it should originate from. The database file is named beezworx_db.db, should the file name be included in the path?
![image](https://user-images.githubusercontent.com/87674745/223571494-970fe402-88fc-4b75-b587-b2cc8d726f17.png)

Using DB Browser, I set up the scheme as suggested in the documentation. 
![image](https://user-images.githubusercontent.com/87674745/223572075-05d6f73a-74b7-4b7b-bda0-dd8759c1a98d.png)

I have went through a couple of the demo files looking for an example project with an already setup sqlite stream but couldnt find an example. 

If I click the terrain dropdown menu in the scene editor, and click regenerate,  the statistics at the bottom change values as if something is being generated, but no mesh is ever showed and no save is ever made to the database.

is there a tutorial/example  online I can refer to?  I have set up and used sqlite for purposes other than voxeltools, but I believe I'm missing/overlooking something.  Can you point me in the right direction?

**Comment by LVLRGames** [comment]
Scratch that, it would not work because I did not append the extension the path.

The correct syntax for the db folder below was : "saves/db/bw_db.sqlite" with no leading slash
![image](https://user-images.githubusercontent.com/87674745/223607352-19e4d7b0-735b-4fa6-a693-e88ba75a8052.png)

**Comment by Zylann** [comment]
So is it solved?

**Comment by LVLRGames** [comment]
Yes, it generates terrain when there is  VoxelStreamSqlite in the stream field
